contracts/lib/MerklePatriciaProof.sol

```diff
@@ -19,11 +19,11 @@ library MerklePatriciaProof {
      */
     function verify(
         bytes32 value,
-        bytes calldata encodedPath,
-        bytes calldata rlpParentNodes,
+        bytes memory encodedPath,
+        bytes memory rlpParentNodes,
         bytes32 root
     )
-        external
+        internal
```

Comment:
We have to make the library linkable instead of deploying it every time. So, the type is changed to "internal" in order to make it linkable so that it can be embedded in the contract directly.